lib/isafw/isaplugins/ISA_cve_plugin.py

```diff
@@ -183,7 +180,7 @@ def process_report_type(self, rtype):
             flog.write("Args: " + args)
         try:
             popen = subprocess.Popen(
-                args, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+                args, shell=True, env=os.environ, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Is it working now using os.environ? I remember it wasn't working in past and I had to go with all this proxy passing etc. because of it not working. Is it the magic of bb.utils.export_proxies(d) now that make it happen?